lib/film_people_list.py

```diff
@@ -0,0 +1,29 @@
+#TODO wstawić później tę funkcję do pliku feature_creators i usunąć ten plik
+import pyspark.sql.functions as f
+from pyspark.sql import DataFrame
+from typing import Union, List, Optional
+
+def people_film_merge_columns(df: DataFrame, film_id: str, add_column: Optional[bool] = False) -> Union[DataFrame,List[str]]:
+  """
+  Funkcja  dla danej tabeli i id filmu zwraca listę z id ludzi, którzy współtworzyli 
+  daną produkcję (jeśli add_column = False) lub wstawia nową kolumnę do tabeli
+  z wierszem zawierającym tę listę (jeśli add_column = True).
+  """
+  assert df.filter(df.id == film_id).rdd.isEmpty() == False
```

Comment:
Sprawdziłbym czy to zadziała. Nie będziemy wtedy korzystać z RDD
```suggestion
if df.filter(df.id == film_id).count() == 0:
    raise AssertionError
```

Reply:
zmieniłem tę asercję